pkg/tracer/tracer_linux.go

```diff
@@ -579,12 +579,18 @@ func (t *LinuxTracer) getMap(mapName string) (*bpflib.Map, error) {
 func (t *LinuxTracer) dispatchPerfEvent(event *common.PerfEvent) {
 	t.tcpConnInsightsLock.Lock()
 	defer t.tcpConnInsightsLock.Unlock()
-
+	connection := common.ConnTuple{}
 	if event.HTTPResponse != nil {
-		logger.Tracef("http response: %v", event.HTTPResponse)
+		if event.HTTPResponse.Connection.IPV4Connection.Lport == 0 {
+			logger.Warnf("http response ipv6: %v", event.HTTPResponse.Connection.IPV6Connection)
```

Comment:
probably, not "warning", but "trace" or "debug"

Reply:
This was your code :laughing: . I'll change to trace.